traversal/src/test/scala/overflowdb/traversal/RepeatTraversalTests.scala

```diff
@@ -202,15 +202,18 @@ class RepeatTraversalTests extends WordSpec with Matchers {
     test(centerTrav.repeat(_.sideEffect(traversedNodes.addOne).out).l)
     test(centerTrav.repeat(_.sideEffect(traversedNodes.addOne).out)(_.breadthFirstSearch).l)
 
-    withClue("for reference: this behaviour is adapted from tinkerpop") {
-      import org.apache.tinkerpop.gremlin.process.traversal.dsl.graph.__.__
-      import org.apache.tinkerpop.gremlin.process.traversal.{Traverser, Traversal => TPTraversal}
-      test(
-        __(center).repeat(
-          __().sideEffect { x: Traverser[Thing] => traversedNodes += x.get }
-            .out().asInstanceOf[TPTraversal[_, Thing]]
-        ).toList.asScala)
-    }
+//    withClue("for reference: this behaviour is adapted from tinkerpop") {
```

Comment:
Maybe a comment if this is going to be brought back or if this is gone forever?

Reply:
added a comment